USAGE.md

```diff
@@ -109,13 +109,21 @@ Note that multiple citations must be semicolon separated.
 Be careful not to cite the same study using identifiers from multiple sources.
 For example, the following citations all refer to the same study, but will be treated as separate references: `[@doi:10.7717/peerj.705; @pmcid:PMC4304851; @pmid:25648772]`.
 
+Citation keys must adhere to the syntax described in the [Pandoc manual](https://pandoc.org/MANUAL.html#citations):
```

Comment:
The text here explains the current behavior well, and the online regex is an especially helpful resource.  Because this is a breaking change, should we more prominently note that as of `<date>` the Manubot citation behavior changed?  I don't know how many authors besides us regularly update rootstock.  I imagine they could be confused, however, if they update and a previously working manuscript starts to fail due to the citation key syntax changes.